angularjs/angular.d.ts

```diff
@@ -1807,7 +1807,6 @@ declare namespace angular {
         controller?: string | Injectable<IControllerConstructor>;
         controllerAs?: string;
         /**
-         * @deprecated
```

Comment:
This deprecation warning doesn't say that `bindToController` is deprecated.  `bindToController`  is here to stay.